File: python/day12/AudioTVParent.py
'''
Created on 2023. 8. 17.

@author: LG
'''
class AudioTVParent:
    def __init__(self, power,volumn):
        self.power = power
        self.volumn = volumn
        
    def Switch(self, onOff):
        self.power = onOff
        
    def set_volumn(self, vol):
        self.volumn = vol

File: day12/src/day123/AudioTVParent.java
package day123;

public class AudioTVParent {
	protected boolean power;
	protected int volumn;
	
	public AudioTVParent() {}
	public AudioTVParent(boolean power, int volumn) {
		this.power = power;
		this.volumn = volumn;
	}
	
	public boolean getPower() {return power;}
	
	public void setPower(boolean power) {
		this.power = power;
	}
	
	public int getVolumn() { return volumn;}
	
	public void setVolumn(int volumn) {
		this.volumn = volumn;
	}
}
